model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/solve_nonhydro.py

```diff
@@ -497,14 +495,6 @@ def __init__(
         )
         self._allocate_local_fields()
         self._determine_local_domains()
-        # TODO (magdalena) vertical nesting is only relevant in the context of
-        #      horizontal nesting, since we don't support this we should remove this option
-        self.l_vert_nested: bool = False
```

Comment:
@muellch It seems like `l_vert_nested` is passed through the `py2f` wrappers. Should we keep this in the signatures for backward compatibility or shall I try to remove this from everywhere?

Reply:
No we should remove it everywhere. We do not care about vertical nesting.
If you need help removing it on the icon-exclaim side, I can help.

Reply:
After discussion with Christoph we decided to keep it in the wrapper because it needs to be passed by the icon Fortran world. If it's passed as `True` then `NonHydrostaticConfig` returns an error because it's not supported by `icon4py`

Reply:
Perfect.